clients/go/compat.go

```diff
@@ -72,16 +78,40 @@ func PackCall(msg ethereum.CallMsg, cipher Cipher) (*ethereum.CallMsg, error) {
 	return &msg, nil
 }
 
+type rsvSigner struct {
+	sign SignerFn
+}
+
+func (s rsvSigner) SignRSV(digest [32]byte) ([]byte, error) {
+	return s.sign(digest)
+}
+
 // PackSignedCall prepares `msg` in-place for being sent to Sapphire. The call will be end-to-end encrypted and a signature will be used to authenticate the `from` address.
-func PackSignedCall(msg ethereum.CallMsg, cipher Cipher, sign SignerFn, chainID big.Int, leash Leash) (*ethereum.CallMsg, error) {
-	dataPack, err := NewDataPack(sign, chainID.Uint64(), msg.From[:], msg.To[:], 0, msg.GasPrice, msg.Value, msg.Data, leash)
+func PackSignedCall(msg ethereum.CallMsg, cipher Cipher, sign SignerFn, chainID big.Int, leash *evm.Leash) (*ethereum.CallMsg, error) {
+	if msg.Gas == 0 {
+		msg.Gas = DefaultGasLimit // Must be non-zero for signed calls.
```

Comment:
So even `1` will work right? Just confirming as we don't use `DefaultGasLimit` otherwise.

Reply:
👍 I thought this was overriding the the `msg.Gas`, but it is not.